### DISCUSSION TITLE
Don't know where to put this...

### DISCUSSION THREAD

**Discussion by geologistluna** [CaptainBobSim/The-Cessna-172-Project-V3#225] [Q&A] [answered]
I've been going through your project and am quite impressed by it.  Its an amazing feat to have done this to this point, however, I am starting to find small errors. One such error is a mislabeled or duplicate .stl which is confusing.  The specific file I am speaking of is "The-Cessna-172-Project-V3/Section 2 - Component Library and Structure/2-5 Instruments/SMALL_EGT Indicator/9g Servo/STLs/Display Backplate.stl" which appears to be a copy of the file "Servo Motor plate.stl" and "Servo SG90 Horn.stl".

Of course I may be completely wrong, and off my rocker. Thank you!

**Comment by CaptainBobSim** [comment, marked as answer]
I deleted this, as I couldn't find a difference. @Kovinator let me know if this is not the case! 

Keep letting me know when you find issues, and feel free to use either the issues or discussions!
